tmt/base.py

```diff
@@ -3514,10 +3535,9 @@ def __init__(self, *, data: Optional[_RawLinks] = None):
 
         # TODO: this should not happen with mandatory validation
         if data is not None and not isinstance(data, (str, dict, list)):
-            raise tmt.utils.SpecificationError(
-                "Invalid link specification "
-                "(should be a string, fmf id or list of their combinations), "
-                f"got '{type(data).__name__}'.")
+            # TODO: deliver better key address, needs to know the parent
+            raise tmt.utils.NormalizationError(
+                'link', data, 'string, fmf id or list of their commbinations')
```

Comment:
Typo: double 'm'

Reply:
Fixed in 9529a776134c84fe8c7428edbfe79f2eb70cfaa4